src/DssSpell.sol

```diff
@@ -60,222 +58,192 @@ contract DssSpellAction is DssAction {
-    uint256 internal constant SEVEN_PCT_RATE               = 1000000002145441671308778766;
-    uint256 internal constant SEVEN_PT_TWO_FIVE_PCT_RATE   = 1000000002219443553326580536;
-    uint256 internal constant SEVEN_PT_FIVE_PCT_RATE       = 1000000002293273137447730714;
-    uint256 internal constant SEVEN_PT_SEVEN_FIVE_PCT_RATE = 1000000002366931224128103346;
-    uint256 internal constant EIGHT_PT_TWO_FIVE_PCT_RATE   = 1000000002513736079215619839;
-    uint256 internal constant NINE_PCT_RATE                = 1000000002732676825177582095;
-
-    // ---------- Contracts ----------
-    GemAbstract internal immutable MKR = GemAbstract(DssExecLib.mkr());
-    VatAbstract internal immutable vat  = VatAbstract(DssExecLib.vat());
-
-    // ---------- Bug Bounty Payout ----------
-    address internal constant IMMUNEFI_COMISSION              = 0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18;
-    address internal constant IMMUNEFI_USER_PAYOUT_2024_08_08 = 0xA4a6B5f005cBd2eD38f49ac496d86d3528C7a1aa;
-
-    // ---------- Update ClipperMomJob ----------
-    address internal immutable CRON_SEQUENCER          = DssExecLib.getChangelogAddress("CRON_SEQUENCER");
-    address internal immutable CRON_CLIPPER_MOM_JOB    = DssExecLib.getChangelogAddress("CRON_CLIPPER_MOM_JOB");
-    address internal constant CRON_CLIPPER_MOM_JOB_NEW = 0x7E93C4f61C8E8874e7366cDbfeFF934Ed089f9fF;
-
-    // ---------- Aligned Delegate Compensation ----------
-    address internal constant BLUE           = 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf;
-    address internal constant CLOAKY         = 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818;
-    address internal constant BYTERON        = 0xc2982e72D060cab2387Dba96b846acb8c96EfF66;
-    address internal constant JULIACHANG     = 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7;
-    address internal constant ROCKY          = 0xC31637BDA32a0811E39456A59022D2C386cb2C85;
-    address internal constant PBG            = 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2;
-    address internal constant CLOAKY_KOHLA_2 = 0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a;
-    address internal constant CLOAKY_ENNOIA  = 0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b;
-
-    // ---------- RWA001-A Stability Fee Increase ----------
-    address internal immutable MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-
-    // ---------- Spark Proxy Spell ----------
-    // Spark Proxy: https://github.com/marsfoundation/sparklend-deployments/blob/bba4c57d54deb6a14490b897c12a949aa035a99b/script/output/1/primary-sce-latest.json#L2
-    address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
-    address internal constant SPARK_SPELL = 0x85042d44894E08f81D70A2Ae568C09f907297dcb;
+
+    // --- Math ---
+    uint256 internal constant MILLION  = 10 ** 6;
+    uint256 internal constant WAD      = 10 ** 18;
+    uint256 internal constant RAD      = 10 ** 45;
+
+    // ---------- LITE-PSM-USDC-A Phase 1 ----------
```

Comment:
What does `Phase 1` imply here? A lot of the addresses below are unrelated to the Phase 1.

Reply:
fixed in [507acca](https://github.com/makerdao/spells-mainnet/pull/422/commits/507accac1551ef44ce574dc4dff7fccd07ed50dd)